### DISCUSSION TITLE
SFR1M44-DU26 w/flashfloppy not showing up in bios

### DISCUSSION THREAD

**Discussion by mpcpro** [keirf/flashfloppy#938] [General]
Hi.

Got an old Olivetti d33 laptop with a 386, the disk and floppy both died so did a hdd change to cf card and trying to get a gotek sfr1m44-du26 working.

I made a temporary 26 pin 1.5mm pitch ffc to 26 pin 1mm pitch adapter by using aliexpress parts and connected that into a 26 pin 1mm pitch to 34 pin floppy adapter and got a gotek sfr1M44-fu working fine in it flashed with flashfloppy (using the usb to flash this device). Managed to install dos this way. But the only way to get the olivetti  bios to see it was to set a jumper for JC and S0 active.

The problem is the du26 doesn't have a JC jumper so the bios just doesn't detect the floppy. I got the du26 flashed with flashfloppy and I followed that issue 155 and desoldered the three resistors and only got a pin set for s1 but still no luck. I tried using the ff.cfg (in root) where i added ibmpc and even added pin34 to ready but no dice.. Though something strange did happen, my usb device  I used stopped getting detected properly with storage and shows up as an external USB disk with no storage, no way to reformat it. I'm wondering if desoldering the three resistors fried the usb.

Anyone know a way to get these du26 gotek to tell the laptop JC is active? A wire to solder? I know technically the flashflopy ff.cfg should be enough but maybe the config I'm using is wrong. Do we have an example of a config laying around for a ibm compatible laptop?

Sorry if this is the wrong place to troubleshoot.

**Comment by keirf** [comment]
RA, RB, RC correspond to JA, JB, JC on a 34-pin Gotek. So you could reinstate RC with a 0-ohm resistor. However, of course, `interface = ibmpc` in FF.CFG should have the same effect.

**Reply by mpcpro** [reply]
> RA, RB, RC correspond to JA, JB, JC on a 34-pin Gotek. So you could reinstate RC with a 0-ohm resistor. However, of course, `interface = ibmpc` in FF.CFG should have the same effect.

Thanks. I will try to reinstate RC. 

It could be that the interface = ibmpc never got loaded properly because the USBpen never worked again after I tried on my other PC (shows up in windows with no space, no partitions...) I do wonder, desoldering the RA/RB/RC, could that be the reason why the USB died?

Also, how long does flashfloppy use to "bootup" the GOTEK? The BIOS on the Olivetti is REALLY fast and perhaps the gotek does not load the config fast enough to emulate "interface = ibmpc". Perhaps thats why I had to have JC "pinned" on my other GOTEK.

**Comment by keirf** [comment]
Is the stick absolutely dead? Can it be repartitioned? It shouldn't have died in this way -- usually the failure mode is completely toasted stick (not even recognised by Windows) because of feeding it more than 5 volts.

**Reply by mpcpro** [reply]
Windows sees it, but there's no partitions and I am unable to reformat or repartition it, it's basically a pen with nothing there. 

I haven't had a chance to test your suggestions yet. The USB pen in question was rather old but once I get some free time I will try again with another one. Another issue could be that the 26 pin fcc was also "homemade", I had a flat cable, but needed a reversible one so I glued a thin layer of plastic on the other end, removed the hard plastic in the other side and sanded down the contacts so it would be reversible ... I know it's a bad hack, ordered a reversible cable on Ali express but a few weeks out and I'm impatient.

So most likely it's my own fault why it's not working 😂

**Comment by mpcpro** [comment]
Hi, Its user error. Got it all working now. Just had to try a proper reverse cable and not my own "home made" one.
